src/test/app/ValidatorSite_test.cpp

```diff
@@ -97,6 +104,11 @@ class http_sync_server
     {
         error_code ec;
         acceptor_.close(ec);
```

Comment:
Does `close` not already protect us?

> This function is used to close the acceptor. Any asynchronous accept operations will be cancelled immediately.
> A subsequent call to open() is required before the acceptor can again be used to again perform socket accept operations. 

http://www.boost.org/doc/libs/1_64_0/doc/html/boost_asio/reference/basic_socket_acceptor/close/overload1.html